### PR TITLE
Url scan added outputs to the URL command

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/31380

## Description
added outputs to the URL command

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.py [hunk 1]
```diff
@@ -284,6 +311,9 @@ def format_results(uuid):
             dbot_score['Score'] = 0
             dbot_score['Type'] = 'url'
             human_readable['Malicious'] = 'Benign'
+    if 'urlscan' in scan_verdicts:
+        if 'tags' in scan_verdicts['urlscan']:
```
Comment (ShahafBenYakir) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.py:315]:
```suggestion
    if 'urlscan' in scan_verdicts and 'tags' in scan_verdicts['urlscan']:
```

File: Packs/UrlScan/ReleaseNotes/1_0_8.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### urlscan.io
+- - Added outputs to the  **URL** command.
```
Comment (ShahafBenYakir) [inline Packs/UrlScan/ReleaseNotes/1_0_8.md:4]:
```suggestion
- - Added new outputs to the  **URL** command.
```

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml [hunk 3]
```diff
@@ -337,6 +337,21 @@ script:
...
+    - contextPath: URL.FeedRelatedIndicators.value
+      description: Indicators that are associated with the URL
+      type: String
+    - contextPath: URL.FeedRelatedIndicators.type
+      description: the type of the indicators that are associated with the URL
+      type: String
+    - contextPath: URL.Geo.Country
+      description: The URL country
+      type: String
+    - contextPath: URL.RelatedASNs
```
Comment (ShahafBenYakir) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:352]: Is this returned like this from the server? I don't see the RelatedASNs here on code

Comment (moishce) [reply Packs/UrlScan/Integrations/UrlScan/UrlScan.yml]: It was a mistake, I fixed

**Review by ShahafBenYakir — APPROVED** [review]

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml [hunk 4]
```diff
@@ -337,6 +337,24 @@ script:
     - contextPath: URLScan.EffectiveURL
       description: Effective URL of the original URL.
       type: string
+    - contextPath: URL.ASN
+      description: The URL ASN
+      type: String
+    - contextPath: URL.FeedRelatedIndicators.value
+      description: Indicators that are associated with the URL
```
Comment (roysagi) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:344]:
```suggestion
      description: Indicators that are associated with the URL.
```

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml [hunk 5]
```diff
@@ -337,6 +337,24 @@ script:
     - contextPath: URLScan.EffectiveURL
       description: Effective URL of the original URL.
       type: string
+    - contextPath: URL.ASN
+      description: The URL ASN
```
Comment (roysagi) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:341]:
```suggestion
      description: The URL ASN.
```

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml [hunk 6]
```diff
@@ -337,6 +337,24 @@ script:
...
+      type: String
+    - contextPath: URL.FeedRelatedIndicators.value
+      description: Indicators that are associated with the URL
+      type: String
+    - contextPath: URL.FeedRelatedIndicators.type
+      description: the type of the indicators that are associated with the URL
```
Comment (roysagi) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:347]:
```suggestion
      description: the type of the indicators that are associated with the URL.
```

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml [hunk 7]
```diff
@@ -337,6 +337,24 @@ script:
...
+      type: String
+    - contextPath: URL.Geo.Country
+      description: The URL country
```
Comment (roysagi) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:350]:
```suggestion
      description: The URL country.
```

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml [hunk 8]
```diff
@@ -337,6 +337,24 @@ script:
...
+      type: String
+    - contextPath: URL.ASOwner
+      description: The URL AS owner
```
Comment (roysagi) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:353]:
```suggestion
      description: The URL AS owner.
```

File: Packs/UrlScan/Integrations/UrlScan/UrlScan.yml [hunk 9]
```diff
@@ -337,6 +337,24 @@ script:
...
+      type: String
+    - contextPath: URL.Tags
+      description: Tags that are associated with the URL
```
Comment (roysagi) [inline Packs/UrlScan/Integrations/UrlScan/UrlScan.yml:356]:
```suggestion
      description: Tags that are associated with the URL.
```

File: Packs/UrlScan/Integrations/UrlScan/README.md [hunk 10]
```diff
@@ -331,6 +331,30 @@
 <td style="width: 267px;">URLScan.EffectiveURL </td>
 <td style="width: 454px;">Effective URL of the original URL</td>
 </tr>
+<tr>
+<td style="width: 267px;">URL.ASN </td>
+<td style="width: 454px;">The URL ASN</td>
```
Comment (roysagi) [inline Packs/UrlScan/Integrations/UrlScan/README.md:336]:
```suggestion
<td style="width: 454px;">The URL ASN.</td>
```

File: Packs/UrlScan/Integrations/UrlScan/README.md [hunk 11]
```diff
@@ -331,6 +331,30 @@
...
+</tr>
+<tr>
+<td style="width: 267px;">URL.FeedRelatedIndicators.value </td>
+<td style="width: 454px;">Indicators that are associated with the URL</td>
```
Comment (roysagi) [inline Packs/UrlScan/Integrations/UrlScan/README.md:340]:
```suggestion
<td style="width: 454px;">Indicators that are associated with the URL.</td>
```

File: Packs/UrlScan/Integrations/UrlScan/README.md [hunk 12]
```diff
@@ -331,6 +331,30 @@
...
+</tr>
+<tr>
+<td style="width: 267px;">URL.FeedRelatedIndicators.type </td>
+<td style="width: 454px;">The type of the indicators that are associated with the URL</td>
```
Comment (roysagi) [inline Packs/UrlScan/Integrations/UrlScan/README.md:344]:
```suggestion
<td style="width: 454px;">The type of the indicators that are associated with the URL.</td>
```

File: Packs/UrlScan/Integrations/UrlScan/README.md [hunk 13]
```diff
@@ -331,6 +331,30 @@
...
+</tr>
+<tr>
+<td style="width: 267px;">URL.Geo.Country </td>
+<td style="width: 454px;">The URL country</td>
```
Comment (roysagi) [inline Packs/UrlScan/Integrations/UrlScan/README.md:348]:
```suggestion
<td style="width: 454px;">The URL country.</td>
```

File: Packs/UrlScan/Integrations/UrlScan/README.md [hunk 14]
```diff
@@ -331,6 +331,30 @@
...
+</tr>
+<tr>
+<td style="width: 267px;">URL.ASOwner </td>
+<td style="width: 454px;">The URL AS owner</td>
```
Comment (roysagi) [inline Packs/UrlScan/Integrations/UrlScan/README.md:352]:
```suggestion
<td style="width: 454px;">The URL AS owner.</td>
```

File: Packs/UrlScan/Integrations/UrlScan/README.md [hunk 15]
```diff
@@ -331,6 +331,30 @@
...
+</tr>
+<tr>
+<td style="width: 267px;">URL.Tags </td>
+<td style="width: 454px;">Tags that are associated with the URL</td>
```
Comment (roysagi) [inline Packs/UrlScan/Integrations/UrlScan/README.md:356]:
```suggestion
<td style="width: 454px;">Tags that are associated with the URL.</td>
```

**Pull request merged**